infra/Pos/Infra/Constants.hs

```diff
@@ -37,8 +37,9 @@ data InfraConstants = InfraConstants
     -- ^ We consider node as known if it was pinged at most @ccEnhancedMessageTimeout@ sec ago
     , ccEnhancedMessageBroadcast :: !Word
       -- ^ Number of nodes from batch for enhanced bessage broadcast
-    , ccNetworkReceiveTimeout    :: !Int
-      -- ^ Network timeout on `recv` in milliseconds
+    , ccNetworkWaitLogInterval   :: !Int
+      -- ^ Network wait logging interval in seconds
+      --   (logging that some recv/send takes significant amount of time)
```

Comment:
Only about log entry every subsequent `ccNetworkWaitLogInterval` s action continues to run